packages/e2e-tests/tests/account-reordering.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { expect, type Page } from '@playwright/test'
+
+import {
+  getUser,
+  createProfiles,
+  User,
+  loadExistingProfiles,
+  reloadPage,
+  test,
+  deleteAllProfiles,
+  waitForAccountItemsToFinishLoading,
+} from '../playwright-helper'
+
+/**
+ * This test covers the drag-and-drop account
+ * reordering functionality in the AccountListSidebar
+ */
+
+test.describe.configure({ mode: 'serial' })
+
+let existingProfiles: User[] = []
+
+const numberOfProfiles = 3 // Need at least 3 accounts to test reordering
+
+let page: Page
+
+test.beforeAll(async ({ browser, isChatmail }) => {
+  const contextForProfileCreation = await browser.newContext()
+  const pageForProfileCreation = await contextForProfileCreation.newPage()
+  await reloadPage(pageForProfileCreation)
+
+  existingProfiles =
+    (await loadExistingProfiles(pageForProfileCreation)) ?? existingProfiles
+  test.setTimeout(120_000)
+
+  await createProfiles(
+    numberOfProfiles,
+    existingProfiles,
+    pageForProfileCreation,
+    browser.browserType().name(),
+    isChatmail
+  )
+
+  await contextForProfileCreation.close()
+  page = await browser.newPage()
+  await reloadPage(page)
+})
+
+test.afterAll(async ({ browser }) => {
+  await page?.close()
+
+  const context = await browser.newContext()
+  const pageForProfileDeletion = await context.newPage()
+  await reloadPage(pageForProfileDeletion)
+  await deleteAllProfiles(pageForProfileDeletion, existingProfiles)
+  await context.close()
+})
+
+test('basic drag-and-drop account reordering', async () => {
+  // Ensure we have at least 3 accounts for meaningful reordering tests
+  expect(existingProfiles.length).toBeGreaterThanOrEqual(3)
+
+  const userA = getUser(0, existingProfiles)
+  const userB = getUser(1, existingProfiles)
+  const userC = getUser(2, existingProfiles)
+
+  // Wait for account items to be rendered and finish loading
+  const accountItems = page.locator('[data-testid^="account-item-"]')
+  await waitForAccountItemsToFinishLoading(page)
+
+  // Get initial account order
+  const initialOrder = await accountItems.allTextContents()
```

Comment:
One can utilize something similar to

https://github.com/deltachat/deltachat-desktop/blob/e75c0d712323734fd2ee2e8ad97c78e0bc915375/packages/e2e-tests/tests/basic-tests.spec.ts#L200-L205